template/build/core/phing/tasks/deploy.xml

```diff
@@ -99,11 +106,13 @@
 
     <!-- Copy required files from docroot. -->
     <copy todir="${deploy.dir}/docroot" overwrite="true">
-      <fileset dir="${docroot}">
+      <fileset dir="${docroot}" defaultexcludes="false">
```

Comment:
I tried that and it didn't work for me. The other option is to check for ACSF and explicitly include it's .gitignore.

I didn't see any reason we should be ignoring all those files and most do not apply.